lib/Cpdf.php

```diff
@@ -3530,15 +3530,15 @@ private function openFont($font)
      * @throws FontNotFoundException
      */
     function selectFont($fontName, $encoding = '', $set = true, $isSubsetting = true)
-    {
-        if ($fontName === null || $fontName === '') {
-            return $this->currentFontNum;
-        }
-
-        $ext = substr($fontName, -4);
+    {   
+        $fontName = (string) $fontName;
+        $ext = substr("$fontName", -4);
```

Comment:
I think `"$fontNae"` is unnecessary since `(string) $fontName` always returns a string (empty string if `$fontName` is null). Which also means checking for null later is also unnecessary.